packages/app-layout/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "bl-app-layout",
+  "version": "1.0.0",
+  "main": "dist/main.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "../../node_modules/.bin/webpack-dev-server --config ../../webpack.config.js --env.dev --hot --host 0.0.0.0 --env.element=app-layout",
+    "test": "npm run build:test && ../../node_modules/.bin/wct -l firefox",
```

Comment:
`firefox` :)

I would probably regenerated it after rebase completely as I think we have been adding some things to the template...
There is also awaiting https://github.com/wc-catalogue/blaze-elements/pull/202